FeedbackSample/src/main/res/raw/foresee_configuration.json

```diff
@@ -7,11 +7,11 @@
     "feedbackSurveys":[
       {
         "name":"Sample 1",
-        "feedbackId":"q9IvXiloOG"
+        "feedbackId":"AEvfwfp1vY"
       },
       {
         "name":"Sample 2",
-        "feedbackId":"gPzT3hhhSw",
+        "feedbackId":"AEvfwfp1vY",
```

Comment:
This seems to be exactly the same feedback survey as Sample 1. I feel we may want a different feedback to better showcase this feature?

Reply:
@awangvan so looking at the intended use of the second example, this seemed to just be checking whether the survey is enabled and then making the check, in addition to not closing the survey. This second example did not appear to be dependent on the content of the second survey.

There was indeed a second feedback survey used in the original training client, however this survey was not actually that much different from the 1st survey. It used the same header and a very similar question. Since the content of the survey was irrelevant to the functional outcome of the second example, using the checkifenabled and delayed survey close features, I didn't create a second survey and chose to adjust both samples to target the same survey.

If you think it'd be wiser to create a secondary survey for this, I'd be happy to do so and adjust the config file. Lmk what you think